### PR TITLE
stage0: Use btrfs with subvolumes for rootfs

### PR DESCRIPTION
**Pull request by raccube**
- Install snapper

### CODE REVIEW

**Comment by raccube** [comment]
Worth noting that this only installs snapper and doesn't configure it

https://wiki.archlinux.org/title/Snapper#Creating_a_new_configuration

**Comment by raccube** [comment]
Maybe that should be removed from this PR?

File: stage0/01-host_pacstrap.sh [hunk 1]
```diff
@@ -5,5 +5,9 @@ source ./util.sh
 
 export CARCH=armv7l
 info "Bootstrapping" "Arch Linux ARM"
-# Reads in common and platform packages and runs pacstrap with them
-pacstrap -cMGC stage0/pacman.conf "$BUILD_DIR" base
\ No newline at end of file
+# bootstraps the system, installing the base and snapper packages
+pacstrap -cMGC stage0/pacman.conf "$BUILD_DIR" base snapper
```
Comment (trickeydan) [inline stage0/01-host_pacstrap.sh]: Is snapper needed at this stage? It could potentially go in a `02-packages`

**Review by trickeydan — DISMISSED** [review]

**Review by trickeydan — APPROVED** [review]
Nice

**Pull request merged**